library/control/test/ProductFeatures_test.rb

```diff
@@ -158,4 +158,95 @@
       end
     end
   end
+
+  describe "#GetFeature" do
+    let(:scr_root_dir) { File.join(File.dirname(__FILE__), "data") }
+    let(:normal_stage) { false }
+    let(:firstboot_stage) { false }
+
+    before do
+      allow(Yast::Stage).to receive(:normal).and_return(normal_stage)
+      allow(Yast::Stage).to receive(:firstboot).and_return(firstboot_stage)
+    end
+
+    around do |example|
+      change_scr_root(scr_root_dir, &example)
+    end
+
+    context "when features are not initialized yet" do
+      it "initializes them" do
```

Comment:
I would 1) something like `expect(subject).to receive(:InitIfNeeded)` or 2) remove this example. See the next comment to understand why.